src/elements/content-sharing/types.ts

```diff
@@ -0,0 +1,22 @@
+import type { ItemType, StringMap } from '../../common/types/core';
+
+export type ContentSharingV2Props = {
```

Comment:
can we keep the `ComponentProps` type in the component file and all the other types will go into this file?